patternlab/styleguide/source/_patterns/01-atoms/09-media/figure--dataviz.twig

```diff
@@ -0,0 +1,31 @@
+{% if figure.dataviz %}
+  {# Render figure.content, which will be the main content of the figure #}
+  {# Important: we wrap this in an `if` statement to allow consuming systems
+     to either use figure.image or figure.content (if they prefer to externally
+     render figure.content
+   #}
+  {% set align = figure.align|default('left') %}
+  {% set id = figure.dataviz.id %}
+  {# TODO:  SET SIZE VALUES #}
+  {# Set default size to large. #}
+  {% set size = "ma__figure--" ~ figure.dataviz.size|default('large') %}
```

Comment:
I think the `size` should be a property of `figure`, given that we will want these same sizes to apply to images and other things in the future, right?  Also, you're formulating the class inside of the `{% if figure.dataviz %}` block.  Keep in mind that Drupal will not use `figure.dataviz` - it will use `figure.content` instead, so we need to move size and class calculation outside of the `if` statement here.

Reply:
Implemented in #971.